src/OpenTelemetry.Sampler.AWS/FallbackSampler.cs

```diff
@@ -20,19 +20,25 @@ namespace OpenTelemetry.Sampler.AWS;
 
 internal class FallbackSampler : Trace.Sampler
 {
-    private static readonly Trace.Sampler AlwaysOn = new AlwaysOnSampler();
-
+    private readonly Trace.Sampler reservoirSampler;
+    private readonly Trace.Sampler fixedRateSampler;
     private readonly Clock clock;
 
     public FallbackSampler(Clock clock)
     {
         this.clock = clock;
+        this.reservoirSampler = new ParentBasedSampler(new RateLimitingSampler(1, clock));
+        this.fixedRateSampler = new ParentBasedSampler(new TraceIdRatioBasedSampler(0.05));
```

Comment:
This is the fallback sampler, only to be used when rules are either not fetched yet or they are expired. When the rules are present, the sampling rates from those rules will be used.

Reply:
In Java, don't we fall back to the localized rules (from config) instead of hardcoded values?

Is this different behavior from Java or am I mistaken?

If it is different, then are we sure we want it to be different?

Reply:
We have the fallback localized rules in X-Ray SDKs. We chose not do those in the OTel SDKs because there not a lot of customers customizing those local rules. They just default to 5%, so that's why instead of having the local rules, we have here a fallback sampler.